openstack_exporter/collectors/cinderbackend.py

```diff
@@ -379,4 +384,91 @@ def collect(self):
                 'Cinder pool custom attribute netapp_fqdn',
                 {'netapp_fqdn': netapp_fqdn},
                 shard_name, backend, pool_name
-            )
\ No newline at end of file
+            )
+
+    def collect(self):
+        """This is the collector for cinder backends.
+
+        This calls the cinder scheduler api to get all the pool stats
+        that the scheduler sees.  If a backend is down and the stats aren't
+        being collected in the driver for that backend, then the scheduler will
+        not have any stats.  The collected has an expected list of backends
+        that it wants to see coming from the scheduler API.  The execpted
+        backends is configurable in the config file.  See the sample config
+
+        For each expected backend the collector will report stats for it.
```

Comment:
typo: sentence starts with "For" and has "for it" at the end. imho get rid of "for it"

Reply:
done